setup.py

```diff
@@ -286,7 +286,10 @@ def find_proj_version_by_program(conda=None):
             proj_includes = proj_includes.decode()
             proj_clibs = proj_clibs.decode()
 
+        if proj_includes.startswith('-I'):
```

Comment:
This only handles the case of the first path starting with `-I`, but what if there's more than one? You need to do this on each element of the list.